### PR TITLE
faq: Add FAQ page with verification info

### PR DESCRIPTION
**Pull request by jameswestman**

### CODE REVIEW

File: frontend/pages/faq.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      <div className="flex flex-col px-[5%] text-justify md:px-[20%] 2xl:px-[30%]">
+        <h1>Frequently Asked Questions</h1>
+
+        <h2>Verified Apps</h2>
+
+        <h3>What does the checkmark under an app name mean?</h3>
+        <p>
+          Some apps have a checkmark on the app page under the developer name.
+          This means the app is published on Flathub by its original developer
+          or a third party approved by the developer. Some apps are published by
```
Comment (razzeee) [inline frontend/pages/faq.tsx:22]: I think this reads hard/misleading. A break/paragraph after the sentence ended might help? 
```suggestion
          or a third party approved by the developer.</p><p> Some apps are published by
```

File: frontend/pages/faq.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import { NextSeo } from "next-seo"
+import Link from "next/link"
+
+const Faq = (): JSX.Element => {
+  const { t } = useTranslation()
+
+  return (
+    <>
+      <NextSeo title={t("faq")} description={t("faq-description")} />
+      <div className="flex flex-col px-[5%] text-justify md:px-[20%] 2xl:px-[30%]">
```
Comment (razzeee) [inline frontend/pages/faq.tsx:13]: I don't think we have done percentages in paddings anywhere else

**Comment by razzeee** [comment]
We probably want to link to this somewhere?

Also, should we translate the main text?

**Comment by barthalion** [comment]
We shouldn't probably call it FAQ, just "What are verified apps?", and yeah, possibly link it also at https://beta.flathub.org/apps/collection/verified somehow?

**Comment by barthalion** [comment]
Yeah. Let's make it a section of the about page for now.

**Comment by jameswestman** [comment]
Ok, it's part of the About page now.

**Pull request merged**